pyscriptjs/src/pyodide.ts

```diff
@@ -1,4 +1,84 @@
-import type { loadPyodide } from 'pyodide';
+import { Runtime } from './runtime';
+import { getLastPath, inJest } from './utils';
+import type { PyodideInterface } from './runtime';
+import { loadPyodide } from 'pyodide';
+// eslint-disable-next-line
+// @ts-ignore
+import pyscript from './python/pyscript.py';
 
-// The current release doesn't export `PyodideInterface` type
-export type PyodideInterface = Awaited<ReturnType<typeof loadPyodide>>;
+export class PyodideRuntime extends Runtime {
+    src = 'https://cdn.jsdelivr.net/pyodide/v0.21.1/full/pyodide.js';
+    name = 'pyodide-default';
+    lang = 'python';
+    interpreter: PyodideInterface;
+    globals: any;
+
+    async loadInterpreter(): Promise<void> {
+        console.log('creating pyodide runtime');
+        // eslint-disable-next-line
+        // @ts-ignore
+        let extraOpts: any = {}
+        if (inJest()) {
+          extraOpts = {indexURL: [process.cwd(), 'node_modules', 'pyodide'].join('/') }
+	}
+        this.interpreter = await loadPyodide({
+            stdout: console.log,
+            stderr: console.log,
+            fullStdLib: false,
```

Comment:
Would be great to allow configuration of this and other things through the PyScript App/Runtime configuration  itself. (Can be in a follow up PR, but we should open an issue for this)

Reply:
noted.